x-pack/test_serverless/README.md

```diff
@@ -209,9 +209,45 @@ node scripts/functional_test_runner.js --config test_serverless/api_integration/
 ## Run tests on MKI
 There is no need to start servers locally, you just need to create MKI project and copy urls for Elasticsearch and Kibana. Make sure to update urls with username/password and port 443 for Elasticsearch. FTR has no control over MKI and can't update your projects so make sure your `config.ts` does not specify any custom arguments for Kibana or Elasticsearch. Otherwise, it will be ignored. You can run the tests from the `x-pack` directory:
 ```
-TEST_CLOUD=1 TEST_ES_URL="https://elastic:PASSWORD@ES_HOSTNAME:443" TEST_KIBANA_URL="https://elastic:PASSWORD@KIBANA_HOSTNAME" node scripts/functional_test_runner --config test_serverless/api_integration/test_suites/search/config.ts --exclude-tag=skipMKI
+TEST_CLOUD=1 TEST_CLOUD_HOST_NAME="CLOUD_HOST_NAME" TEST_ES_URL="https://elastic:PASSWORD@ES_HOSTNAME:443" TEST_KIBANA_URL="https://elastic:PASSWORD@KIBANA_HOSTNAME" node scripts/functional_test_runner --config test_serverless/api_integration/test_suites/search/config.ts --exclude-tag=skipMKI
 ```
 
+Steps to follow to run on QA environment:
+- Go to [QA Cloud Console](https://console.qa.cld.elstc.co/) and create a project.
```

Comment:
__nit:__ let's replace `console.qa.cld.elstc.co` with `[PROJECT_API_HOST]` or `[CLOUD_HOST_NAME]`, or something along these lines here and in all notes below. I see we try not to mention these URLs explicitly in public repos. I personally don't see any issue with that since it's already publicly available, but let's omit them for now for the sake of consistency.

Reply:
Done ✅